### PR TITLE
Fix tls_key_file check

### PR DESCRIPTION
**Pull request by Praying**
`!tls_cert_file.is_empty()` in src/bin/server.rs is duplicated，remove it.

### CODE REVIEW

**Review by Praying — COMMENTED** [review]
it seems ok.

File: src/bin/server.rs [hunk 1]
```diff
@@ -110,7 +110,7 @@ pub async fn main() -> tidis::Result<()> {
         listener = Some(TcpListener::bind(&format!("{}:{}", &listen_addr, port)).await?);
     }
 
-    if tls_port != "0" && !tls_cert_file.is_empty() && !tls_cert_file.is_empty() {
+    if tls_port != "0" && !tls_cert_file.is_empty()  {
```
Comment (sunxiaoguang) [inline src/bin/server.rs:113]: I guess it suppose to be guard for both `tls_key_file` and `tls_ca_cert_file`.

**Review by sunxiaoguang — COMMENTED** [review]
Good catch, and thanks for your contribution!

I left a minor comment, PTAL.

**Comment by yongman** [comment]
@Praying @sunxiaoguang the `tls_ca_cert_file` can be empty if `tls_auth_client` is not enabled, so `tls_ca_cert_file` should not be checked here.

**Review by yongman — APPROVED** [review]
LGTM

**Pull request merged**